<?xml version="1.0"?>
<launch> 
   <arg name="decimation" default="1"/> <!-- 1 -->
   <arg name="rate" default="5"/>
   <arg name="table_per" default="0.01"/>
   <arg name="stype" default="single"/>
   <arg name="use_beacons" default="false"/>

   <arg name="detect_box" default="true"/>
   <arg name="detect_obj" default="true"/>

   <arg name="camera_frame" default="camera_rgb_optical_frame"/>
   <arg name="image_topic" default="/camera/rgb/image_rect_color_drop"/>
   <arg name="cloud_topic" default="/camera/depth_registered/points"/>
   <arg name="ref_frame" default="/base" />

   <node name="drop_node" pkg="topic_tools" type="drop" args="/camera/rgb/image_rect_color 3 4 /camera/rgb/image_rect_color_drop"/>

   <node name="ball_track" pkg="mdb_baxter_detection" type="ball_track" required="true" output="screen" if="$(arg detect_obj)" > <!-- launch-prefix="valgrind"-->
       <remap from="/image_color" to="$(arg image_topic)" />

       <param name="table_per" type="double" value="$(arg table_per)"/>
   </node>

</launch>


